src/components/file-list/file-title/file-title.utils.js

```diff
@@ -0,0 +1,9 @@
+export const removeFileNameExtension = (fileName) => {
+  const fileExtensionExp = /.(mp3|wav|ogg)$/;
+  if (fileExtensionExp.test(fileName)) {
+    const lastDotIndex = fileName.lastIndexOf('.');
+    return fileName.substring(0, lastDotIndex);
+  }
+
+  return fileName;
+};
```

Comment:
```jsx
const FILE_EXTENSION_REGEX = /.(mp3|wav|ogg)$/;

export const getFileNameWithoutExtension = (fileName) => {
  const [ title ] = fileName.match(FILE_EXTENSION_REGEX);
  return title || fileName;
};
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

Reply:
After experimenting and reading, finds out we can do something simpler.

```
const FILE_EXTENSION_REGEX = /.(mp3|wav|ogg)$/;

export const getFileNameWithoutExtension = (fileName) =>
  fileName.replace(FILE_EXTENSION_REGEX, '');
```